source/_integrations/kira.markdown

```diff
@@ -10,12 +10,16 @@ ha_release: 0.45
 ha_domain: kira
 ---
 
-The `kira` integration is the main integration to integrate Keene Electronics IR over IP [Kira](https://www.keene.co.uk/keene-ir-anywhere-single-worldwide.html) modules with Home Assistant.
+The `kira` integration is the main integration to integrate Keene Electronics IR over IP [Kira](https://k2audio.co.uk/collections/ip-and-internet-control) modules with Home Assistant.
 
 There is currently support for the following device types within Home Assistant:
 
-- Remote
-- Sensor
+- Remote (emit an IR code when instructed by Home Assistant)
+- Sensor (trigger Home Assistant to do something when a particular IR signal is received)
+
+Some models (original Kira and Kira128) can be configured to act as either a sensor or as a remote. They are also able act as both when set to Standalone mode. The wireless models are hardware specific so the receiver can only be integrated as a sensor and the transmitter can only be integrated as a remote. 
+
+If you are using two or more Kira devices for point to point IR transfer across your network they can continue to perform this function whilst also acting as a sensor or remote for Home Assistant.
 
 ### Example Configuration
```

Comment:
The first paragaph should be called just "Configuration" and is a level 2 header

```suggestion
## Configuration
```